config/sites/icsa.uiowa.edu/config_split.config_split.site.yml

```diff
@@ -25,6 +19,9 @@ graylist:
   - system.menu.services
   - views.view.projects
   - views.view.reports
+  - 'field.storage.node.field_project_*'
+graylist:
+  - core.date_format.medium_date
```

Comment:
This is a complete split as medium_date is custom and not part of default

Reply:
Forgot to push up this change. D'oh!